### PR TITLE
Silent/Quite mode implementation

### PR DESCRIPTION
**Pull request by Rishabh04-02**
#2

### CODE REVIEW

**Comment by Rishabh04-02** [comment]
#2 
Please review.

**Comment by fr05t1k** [comment]
Please format your code by [PSR-2](http://www.php-fig.org/psr/psr-2/)

**Comment by Rishabh04-02** [comment]
Please review.

**Comment by fr05t1k** [comment]
Please check my previous comments.

**Comment by Rishabh04-02** [comment]
Can you point out some lines, that will help me re-formatting the code efficiently.

**Comment by fr05t1k** [comment]
[here](https://github.com/fr05t1k/codeception-progress-reporter/pull/3/files/f49882e2a9d560602cea110ff20c89189574e58d#diff-c186f1849483829c56faaf880d0c6c49)

File: src/ProgressReporter.php [hunk 1]
```diff
@@ -57,16 +57,29 @@ class ProgressReporter extends Extension
...
+            $this->_reconfigure(['settings' => ['silent' => true]]);
+            $this->standardReporter = new Console($this->options);
+            ProgressBar::setFormatDefinition(
+                'custom',
+                "Current test: <options=bold>%file%</>\n".
+                "<fg=green>Success: %success%</> <fg=yellow>Errors: %errors%</> <fg=red>Fails: %fails%</>\n" .
+                "<fg=cyan>[%bar%]</>\n%current%/%max% %percent:3s%% %elapsed:6s%/%estimated:-6s% %memory:6s%\n"
+            );
+        }
+        else if($this->options['silent'] == true)
```
Comment (kumarmj) [inline src/ProgressReporter.php]: `else if` must be in above line, & braces must be in front after the condition
image

**Comment by Rishabh04-02** [comment]
the link you've mentioned in the above comment is pointing to the old code commit.
Please review the latest code [here](https://github.com/fr05t1k/codeception-progress-reporter/pull/3/files)

**Comment by fr05t1k** [comment]
Already implemented in https://github.com/fr05t1k/codeception-progress-reporter/pull/4

**Pull request closed without merging**